packages/@aws-cdk/aws-elasticsearch/test/integ.elasticsearch.expected.json

```diff
@@ -522,6 +523,7 @@
           "DedicatedMasterEnabled": false,
           "InstanceCount": 1,
           "InstanceType": "r5.large.elasticsearch",
+          "WarmEnabled": false,
```

Comment:
What happens if this key is passed to clusters of a version that don't support `UltraWarm` - will this be ignored? I suspect it might fail.

Reply:
I did one test and it didn't fail. However, I changed it to set the warmEnabled property only if it's enabled. Thus, we shouldn't get any unexpected issues.

Reply:
👍